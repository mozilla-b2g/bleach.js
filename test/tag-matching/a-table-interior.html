<table>
  <tr>
    <td colspan="3" style="">
      <a href="http://ex.co/a" class="cta_btn">
        <table cellspacing="0" cellpadding="0" width="100%" bgcolor="#4c649b" class="btn_confirm">
          <tr>
            <td height="14" colspan="3">
              &nbsp;</td>
          </tr>
          <tr>
            <td width="38">
              &nbsp;</td>
            <td width="100%">
              <a href="http://ex.co/a">
                <center>
                  <font size="4">
                    <span>
                      Confirm&nbsp;Your&nbsp;Account</span>
                  </font>
                </center>
              </a>
            </td>
            <td width="38">
              &nbsp;</td>
          </tr>
          <tr>
            <td height="14" colspan="3">
              &nbsp;</td>
          </tr>
        </table>
      </a>
    </td>
  </tr>
</table>
