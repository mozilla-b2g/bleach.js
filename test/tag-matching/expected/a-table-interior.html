<table>
  <tr>
    <td colspan="3" style="">
      <a ext-href="http://ex.co/a" class="cta_btn moz-external-link">
        <table cellspacing="0" cellpadding="0" width="100%" bgcolor="#4c649b" class="btn_confirm">
          <tr>
            <td height="14" colspan="3">
              &nbsp;</td>
          </tr>
          <tr>
            <td width="38">
              &nbsp;</td>
            <td width="100%">
              <a ext-href="http://ex.co/a" class="moz-external-link">
                <center>
                  <font size="4">
                    <span>
                      Confirm&nbsp;Your&nbsp;Account</span>
                  </font>
                </center>
              </a>
            </td>
            <td width="38">
              &nbsp;</td>
          </tr>
          <tr>
            <td height="14" colspan="3">
              &nbsp;</td>
          </tr>
        </table>
      </a>
    </td>
  </tr>
</table>
